### PR TITLE
Iio fixes to make it work without the need to add network files

### PR DESCRIPTION
**Pull request by mchindri**

### CODE REVIEW

**Review by buha — APPROVED** [review]

File: tools/scripts/src_model.mk [hunk 1]
```diff
@@ -28,3 +28,5 @@ IGNORE_FILES += $(PLATFORM_DRIVERS)/irq.c
 # LIBRARIES += mbedtls
 LIBRARIES += iio
 
+# If the variable is set to y then iio network backend will be enabled
+ENABLE_IIO_NETWORK = y
```
Comment (amiclaus) [inline tools/scripts/src_model.mk:32]: nitpick: add new line at the end of the file

**Review by amiclaus — APPROVED** [review]
One nitpick on my side.

The rest LGTM

**Review by adrimbarean — APPROVED** [review]
Looks good to me.

**Comment by mchindri** [comment]
v2:
 - fix asyle
 - add new line
 - add new commit (add IIO_SUPPORT flag when iio library is added)

**Review by dbogdan — APPROVED** [review]

**Pull request merged**